### ISSUE TITLE
All media players button are non-descriptive and labelled "play" (WCAG 2.4.6) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#202]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
**Page:**
Course Overview page
Lesson 1: Overview of electronic Passport Next Generation
Lesson 2: Domestic Print
Lesson 4: Manual QA
Lesson 5: Reports

**Description:**
There are two audio controls that play different content but do not properly distingush their content and simply have "Play": 
   • Welcome!
   • Course Structure

Each button should be descriptive: "Play Welcome audio", "Play Conclusion audio"

Please add a more descriptive name for the audio so that users can distingush their content before playing them.
*******************************************************************************
